tools/cloudflow-descriptor-generator/src/main/scala/akka/cloudflow/Main.scala

```diff
@@ -0,0 +1,30 @@
+package akka.cloudflow
+
+import com.typesafe.config.{ ConfigFactory, ConfigRenderOptions }
+
+import java.net.URL
+import scala.collection.JavaConverters._
+
+object Main {
+
+  def main(args: Array[String]): Unit = {
+    if (args.size != 1) {
+      Console.err.println("Need exactly one argument that is the configuration file")
```

Comment:
What is the format of this file?

Reply:
We probably want to tune the format of this a bit. Is it 
``
{ 
  projectId = ..., 
  classpath = [], 
  dockerImageName = ...
}
```
?

Reply:
I guess I should see this `Main` as a POC at this point..

Reply:
yes, do you want to remove it?

Reply:
no I think it is OK.